crates/cheatcodes/src/utils.rs

```diff
@@ -145,6 +146,17 @@ impl Cheatcode for ensNamehashCall {
     }
 }
 
+impl Cheatcode for randomUintCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self {} = self;
+        // Use thread_rng to get a random number
+        let mut rng = rand::thread_rng();
+        let num: [u64; 4] = rng.gen();
+        let random_number = U256::from_limbs(num);
```

Comment:
U256::random()

Reply:
Don't think that's implemented in alloy_primitives

Reply:
yeah it's not , this is fine

Reply:
it should be implemented if "rand" is enabled, and this becomes let random_number: U256 = rng.gen();